docs/site/Best-practices-with-Loopback-4.md

```diff
@@ -8,21 +8,37 @@ permalink: /doc/en/lb4/Best-practices-with-Loopback-4.html
 summary:
```

Comment:
When I was working on this page to include sublists, I know I came across issues with Jekyll not converting the sublists to html correctly. Could you please manually check if the formatted page renders fine in `loopback.io`?